scripts/reset_lambda_role.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import time
+
+import boto3
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    client = boto3.client('lambda')
+
+    paginator = client.get_paginator('list_functions')
+
+    response_iterator = paginator.paginate()
+    lambda_names = set(config.lambda_names())
+
+    for page in response_iterator:
```

Comment:
Consider using `azul.lambdas.Lambdas.list_lambdas` and moving some/all of this functionality into that module.